docs/usage/code-coverage.mdx

```diff
@@ -7,121 +7,136 @@ description: Pester can generate code coverage metrics to help you understand ho
 
 Code Coverage refers to the percentage of lines of code that are tested by a suite of unit tests. It's a good general indicator of how thoroughly your code has been tested, that all branches and edge cases are working properly, etc. Pester can generate these code coverage metrics for you while it is executing unit tests.
 
-Note: Unlike most of Pester, use of the Code Coverage feature requires PowerShell version 3.0 or later.
-
-To generate Code Coverage metrics, pass one or more values to the `-CodeCoverage` parameter of the `Invoke-Pester` command. The arguments to this parameter may be one of the following:
-
-1. Strings, which refer to the path of the script files for which you want to generate coverage metrics. These strings may contain wildcards.
-2. Hashtables, which give you finer control over which sections of the file are analyzed for coverage. Using these hashtables, you may limit the analysis to specific functions or ranges of lines within a file.
-
-The hashtables may have the following keys:
-
-- `Path` or `p`: Path to the file on disk. This is the only required key in the hashtable, and passing a hashtable which contains only this key is the equivalent of just passing path as a string value to the `-CodeCoverage` parameter. As with passing strings to this parameter, the `Path` key may contain wildcards.
-- `Function` or `f`: The name of a function you wish to analyze within a file. This value may contain wildcards, and any matching functions will be analyzed. If this key is used, `StartLine` and `EndLine` are ignored for this particular hashtable.
-- `StartLine` or `s`: The first line of a range to be analyzed. If this key is used and no corresponding value is assigned to `EndLine`, then the entire remainder of the file starting with `StartLine` will be analyzed.
-- `EndLine` or `e`: The last line of a range to be analyzed. If this key is used and no corresponding value is assigned to `StartLine`, the entire file up to and including `EndLine` will be analyzed.
-
-After `Invoke-Pester` finishes executing the test scripts, Pester will output a coverage report to the console. If you are using `Invoke-Pester`'s `-PassThru` switch, the coverage analysis will also be available on the output object, under its `CodeCoverage` property.
-
-Here are some examples of the various ways the `-CodeCoverage` parameter can be used, and their corresponding output when used with the following files.
-
-```powershell title="CoverageTest.ps1"
```

Comment:
Keep the example files + matching output for the inital demo? Maybe easier to undestand the concept for new users?

Reply:
Old example files integrated.